### ISSUE TITLE
cSpell.json overwritten with commented json on word add.

### ISSUE THREAD

**Issue by will-shaw** [Jason-Rev/vscode-spell-checker#206]
Hopefully I haven't missed a setting somewhere, but is it possible to have the spell checker not write comments into the cSpell.json when you add a word through the command palette?

**Comment by will-shaw** [comment]
@Jason3S Thanks for your reply, unfortunately that doesn't seem to be the case for me. 
I've just tried your suggestion on VSCode stable, as I was currently using Insiders and had a thought that could potentially cause it, but same on Stable too. 

To reproduce:

1. I created the cSpell.json file in .vscode/ manually
2. Then find a word in source code which is incorrect
3. Right click > Add word to workspace dictionary 
4. It adds the word, but all the comments come back.
